MyCryptoHQ/MyCrypto
# [BUGFIX] Fix disabled Schedule button

### Description

There was a problem that when you:
1. Fill address
2. Fill amount of ETH
3. Press 'Send Later' button

You would have 'Schedule transaction' button disabled.

Reason has been setting window size when window size input field was rendered - which is obviously, not the best solution. Since we've moved window size field to Advanced section, it hasn't been rendered - and - the schedule transaction button would remain disabled until toggling + Advanced section.

This PR fixes the problem.

### Changes

* Fix problem with "Schedule Transaction" button staying disabled until clicking "+ Advanced"
* Use just one file and component for Window Size instead of 4

### Steps to Test

1. Fill address
2. Fill amount of ETH
3. Press 'Send Later' button

Expected: "Schedule Transaction" button should be enabled when you have enough funds
